src/Microsoft.TestPlatform.CrossPlatEngine/TestSession/ProxyTestSessionManager.cs

```diff
@@ -20,6 +22,13 @@ namespace Microsoft.VisualStudio.TestPlatform.CrossPlatEngine;
 /// </summary>
 public class ProxyTestSessionManager : IProxyTestSessionManager
 {
+    private enum TestSessionState : int
+    {
+        Success = 0,
```

Comment:
Would it be worth introducing an `unknown` state?

Reply:
To be honest I don't see much value in introducing an `Unknown` state yet, but we'll benefit from it when we'll implement the telemetry revamp proposal (i.e. the one where we create actual classes for telemetry events) so I added it for future proofing. Then we'll need a default value there and that should probably be `Unknown` as you suggested.